src/service/donateService.js

```diff
@@ -0,0 +1,12 @@
+const Donate = require('../model/Donate');
+
+const create = async (donate) => {
```

Comment:
`create` 메서드도 좋지만, 어떤 것을 create 할 지도 명시해주면 코드 가독성에도 더 좋을 것 같네요!
```suggestion
const createDonate = async (donate) => {
```

Reply:
내가 짠 코드 나만 알아본다는 심보 잡혀버렸네요